.gitignore

```diff
@@ -1,4 +1,5 @@
 # Project-specific files
+/.vscode
```

Comment:
Can you remove this line? VS Code info was added to the gitignore in https://github.com/badges/shields/pull/2581/files to ensure we'd exclude settings files, but allow for the inclusion of other files (like extensions currently and potentially launch, etc. in the future)